pcawg_metadata_parser/run_me.sh

```diff
@@ -31,6 +31,11 @@ cd ../s3-transfer-operations/
 git pull
 cd $DIR
 
+echo update ceph-transfer-operations git submodule
+cd ../ceph_transfer_ops/
+git pull
```

Comment:
Yes. It was already set up.

-Linda




On Oct 6, 2015, at 10:48 AM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


In pcawg_metadata_parser/run_me.sh<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/37#discussion_r41273653>:

> @@ -31,6 +31,11 @@ cd ../s3-transfer-operations/
>  git pull
>  cd $DIR
>
> +echo update ceph-transfer-operations git submodule
> +cd ../ceph_transfer_ops/
> +git pull


please make sure this new git submodule is set up and initiated on the pancancer.info server, otherwise, it will fail.

—
Reply to this email directly or view it on GitHub<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/37/files#r41273653>.